impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -50,11 +50,11 @@ static ShaderType GetShaderType(RuntimeUniformType type) {
   }
 }
 
-static std::shared_ptr<ShaderMetadata> MakeShaderMetadata(
+static ShaderMetadata MakeShaderMetadata(
```

Comment:
nit: I think we should move this to `std::unique_ptr<ShaderMetadata>`.  That way we can pass ownership to the Resource.  We could use std::move with ShaderMetadata but that isn't enforced like unique_ptr is.  It's only a nit since it only effects runtime effect's performance.

Reply:
done